packages/experience-builder-sdk/src/core/preview/PrebindingManager.ts

```diff
@@ -0,0 +1,547 @@
+import {
```

Comment:
Would it make sense to keep a minimal version of the prebindingManager that only includes:
* `storePrebindingDefinitions`
* `storeParameterDefinitions`
* `getHoistedParameterByNodeId`

Given that the one in the Web App has already diverged from this one, it just makes things more complicated to keep them in sync.

Reply:
yes, it makes sense 👍

Reply:
I will align it with the latest version from the web app I think